ApplicationTrackingSystem/Controllers/JobPostController.cs

```diff
@@ -17,7 +17,7 @@ public JobPostController(IUnitOfWork unitOfWork)
 
         public IActionResult Index()
         {
-            var jobPosts = _unitOfWork.JobPost.GetAll(); 
+            var jobPosts = _unitOfWork.JobPost.GetAll();
             ViewBag.TodayDate = DateTime.Today.ToString("MMMM dd, yyyy");
```

Comment:
there are multiple jquery options available to use date time in view , do not use viewbag.